src/ast/ddl.rs

```diff
@@ -1291,11 +1406,13 @@ impl fmt::Display for ColumnOption {
                 write!(f, "OPTIONS({})", display_comma_separated(options))
             }
             Identity(parameters) => {
-                write!(f, "IDENTITY")?;
-                if let Some(parameters) = parameters {
-                    write!(f, "({parameters})")?;
-                }
-                Ok(())
+                write!(f, "{parameters}")
+            }
+            Policy(parameters) => {
+                write!(f, "{parameters}")
+            }
+            Tags(tags) => {
+                write!(f, "WITH TAG ({})", display_comma_separated(tags))
```

Comment:
hmm I wonder if its correct to include the `WITH` here, maybe the `Tags` enum variant should rather be called `WithTags` in that case or something similar. But looking at the syntax, the WITH looks optional? If so we should include test coverage for both scenarios, then I'm thinking the variant can look like
```rust
struct TagsColumnOption {
    tags: Vec<Tag>,
    with: bool
}
enum  ColumnOption {
  ...
  Tags(TagsColumnOption)
}

Reply:
Current implementation of similar options of table has no this flag - it is a reason that I skipped it.

I added `with` flag to existing structs and updated tags columns option definition.